.github/workflows/deploy.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ main ]
+    tags: [ 'v[0-9]+.[0-9]+.[0-9]+' ]
+
+env:
+  RSA_PRIVATE_KEY: ${{ secrets.RSA_PRIVATE_KEY }}
+  
+
+jobs:
+  check:
+    outputs:
+      status: ${{ steps.check_env_file.outputs.files_exists }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Check Enc Env file existence
+        id: check_env_file
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "env/.env.dev.template" # "env/.env.${{ github.ref_name }}.enc"
```

Comment:
I'm guessing we will switch to encoded files once we have the rsa_private_key using a process-bp template? Will the actions work till then? Since we won't actually commit values to the .env.dev.template file right?

Reply:
1. True
2. Yes
3. That file is just a dummy since this step's `true` output is being used to determine whether to run deploy or not

As mentioned by you in the doc this only needs 1 env var openai_api_key, and that can be provided currently through the repo secrets, you can check below I'm creating a file `.env` and putting that env var into it. (Currently it's value is just `FAKE_KEY`)